### PR TITLE
Data Prepper 2.8.0 release blog post. 

### LINKED ISSUE #2856: Data Prepper 2.8.0 release blog
**Describe the blog post your would like to write**

We are releasing Data Prepper 2.8.0. We will have a release blog accompany this.

**What is the title of the blog post?**

Announcing Data Prepper 2.8.0

**Who are the authors?**

* Dinu John
* David Venable

**What is the proposed posting date?**

May 15, 2024

### PR DESCRIPTION
**Pull request by dlvenable**
### Description

This is the blog post for releasing Data Prepper 2.8.0. We plan to release on May 16.
 
### Issues Resolved

Resolves #2856.

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.8.0"
+authors:
+- qichen
```
Comment (dlvenable) [inline _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md:5]: Fix alias: https://opensearch.org/community/members/george-chen.html

File: _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+
+
+## Other features
+
+* The `s3` sink can write events to [Amazon S3](https://aws.amazon.com/s3/) objects with data from events in the key paths. This allows you to organize your key prefixes by dates or other metadata. This allows tools such as [Apache Hadoop](https://hadoop.apache.org/) and [Amazon Athena](https://aws.amazon.com/athena/) to more efficiently perform ad-hoc queries on your data.
+* Data Prepper now has an `ndjson` codec you can use to read data from S3 that looks like ndjson. By default, it is far more lenient and does not require newlines between JSON objects.
+* The new `write_json` processor can serialize event data into a JSON string that you can send downstream.
+* The `s3` sink supports bucket ownership validation for more secure interactions with Amazon S3.
+* A new `typeof` operator allows you to write conditions which check the type of a field. You can use this anywhere you use other Data Prepper expressions such as in routes or when conditions.
```
Comment (natebower) [inline _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md:103]:
```suggestion
* A new `typeof` operator allows you to write conditions that check the type of a field. You can use this anywhere you use other Data Prepper expressions, such as in routes or conditions.
```

**Review by natebower — COMMENTED** [review]
@dlvenable Please see my comments and changes and let me know if you have any questions. Thanks!

File: _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+Each message chunk is stand-alone message such a way that the consumer of the message can start processing it immediately instead of waiting for all chunks to be received.
+This chunking of messages is currently supported only for JSON messages received by http source. Support for large OTEL metrics/traces/logs will be added in a future release.
+
+
+
+
+## Other features
+
+* The `s3` sink can write events to [Amazon S3](https://aws.amazon.com/s3/) objects with data from events in the key paths. This allows you to organize your key prefixes by dates or other metadata. This allows tools such as [Apache Hadoop](https://hadoop.apache.org/) and [Amazon Athena](https://aws.amazon.com/athena/) to more efficiently perform ad-hoc queries on your data.
+* Data Prepper now has an `ndjson` codec you can use to read data from S3 that looks like ndjson. By default, it is far more lenient and does not require newlines between JSON objects.
```
[the remark this answers is not in the document]
Comment (dlvenable) [reply _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md:100]: I've normally see "newline" as a single word for these purposes. Wikipedia has this as well: https://en.wikipedia.org/wiki/Newline

[the remark this answers is not in the document]
Comment (natebower) [reply _posts/2024-05-16-Announcing-Data-Prepper-2.8.0.md]: Ok, works for me 😄

**Review by natebower — APPROVED** [review]
@dlvenable LGTM!

**Comment by pajuric** [comment]
@nateynateynate @krisfreedain - The blog is ready for distribution today.

**Review by nateynateynate — APPROVED** [review]
Looks good with the excerpt in there. 

Don't forget to `bundle exec jekyll serve` so that you can see what your blogs look like before you submit the PR.

**Pull request merged**